api/client/client.ts

```diff
@@ -77,15 +102,15 @@ export class AtatClient implements IAtatClient {
   private readonly client: AxiosInstance;
   private readonly logger: ILogger;
 
-  constructor(authToken: string, cspConfiguration: CspConfigurationItem, logger?: ILogger) {
+  constructor(authToken: string, cspUri: string, logger?: ILogger) {
     this.client = axios.create({
-      baseURL: cspConfiguration.uri,
+      baseURL: cspUri,
       headers: {
         Authorization: `Bearer ${authToken}`,
         "User-Agent": "ATAT CSP Api Client",
         "Content-Type": "application/json",
         "X-Atat-Api-Version": AtatClient.supportedApiVersion,
-        "X-Target-Impact-Level": cspConfiguration.network,
+        // "X-Target-Impact-Level": cspConfiguration.network,
```

Comment:
There is a bug (which I introduced by accident) related to the `X-Provision-Deadline` header, but I wanted to mention it, so it is captured. The deadline should only be added when an `addEnvironment` operation happens, but currently any request that uses the `buildHeaders` method will have the deadline header.

A solution would be to have some kind of check before adding the header in the `buildHeaders` method using a specific property that is unique to the `addEnvironment` request.

Reply:
Adding a TODO for this

Reply:
Actually I fixed it by only setting it if the request actually included a deadline:
```
    if (request.provisionDeadline) {
      headers["X-Provision-Deadline"] = request.provisionDeadline;
    }

```